release-notes/6.0/api-diff/.Net/6.0.0_Microsoft.Win32.md

```diff
@@ -0,0 +1,99 @@
+# Microsoft.Win32
```

Comment:
What changed here that all of this surface area is showing up?

Reply:
Not sure, I have not been keeping track of all the changes made everywhere other than areas I feel like I could benefit from on the changes.